libs/net/services/Config/IngestServiceOptions.cs

```diff
@@ -7,8 +7,19 @@ public class IngestServiceOptions : ServiceOptions
 {
     #region Properties
     /// <summary>
-    /// get/set - An array of media type this service will ingest.
+    /// get/set - An array of media type this service will ingest (comma separated).
     /// </summary>
-    public string[] MediaTypes { get; set; } = Array.Empty<string>();
+    public string MediaTypes { get; set; } = "";
```

Comment:
Arrays are additive, not subtractive, which results in unexpected configuration override behaviour.  Using a string with a comma-separator is easier.